Dockerfile

```diff
@@ -113,18 +113,20 @@ RUN make test
 
 FROM base-fpm AS build-fpm-composer
 
+WORKDIR /var/www/html
+
 ARG COMPOSER_USER
 ARG COMPOSER_PASS
 ARG AS3CF_PRO_USER
 ARG AS3CF_PRO_PASS
 
 COPY --from=composer:2 /usr/bin/composer /usr/bin/composer
 
-WORKDIR /var/www/html
+COPY ./bin/composer-auth.sh ./bin/composer-post-install.sh ./bin/
 
```

Comment:
Can you pop `WORKDIR /var/www/html` on line 132 so `exec` can land us in the right place?

Reply:
Are you ok with it being a few lines up to match what we have on Justice UK? It's on 11 at the mo.

Reply:
Mate, wherever you feel is best.

Reply:
I didn't see it there lol

Reply:
haha, I was trying to simplify things, just made it more complicated 😅